apps/frontend/src/libs/components/date-input/styles.module.css

```
@@ -0,0 +1,124 @@
+:global(.react-calendar) {
```

assign a class name to the calendar wrapper, then we should be able to use it this way, to make it less global
```suggestion
.calendar :global(.react-calendar) {
```